File: interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.entity.Result;
import cn.itcast.core.pojo.seckill.SeckillOrder;

import java.util.List;

public interface SeckillService {
    //秒杀订单列表
    public PageResult findPageAllSeckill(String userName, Integer page, Integer rows, SeckillOrder seckillOrder);

    //取消订单
    public void update(Long id);

}

File: web_user/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findPageAllSeckill=function(page,rows){
		var obj =  $http.get('../seckill/findPageAllSeckill.do?page='+page+'&rows='+rows);
        console.log(obj);
        return obj;
	}


    //取消订单
    this.update=function(id){
        return $http.post('../seckill/update.do?id='+id);
    }

	
});
